strings.go

```diff
@@ -20,7 +20,7 @@ func Stringify(message interface{}) string {
 // stringifyValue was graciously cargoculted from the goprotubuf library
 func stringifyValue(w io.Writer, val reflect.Value) {
```

Comment:
is this only ever used by `Stringify(message interface{})` ? If so, I'd change the type of `w` to be `bytes.Buffer` to make clear why errors are all dropped on the floor.